javascript/features/limits/limits_decider.js

```diff
@@ -150,6 +154,15 @@ export class LimitsDecider extends PlayerEventObserver {
         return null;
     }
 
+   // Processes if the player is present in vehicle and makes sure if the player isn't in one.
+   // This proces is required to avoid uncertain bugs .
+    processVehicleRequirement(player) {
+        if(player.state == Player.kStateVehicleDriver && Player.kStateVehiclePassenger)
+        return LimitsDecision.createRejection(Message.LIMITS_DEATH_MATCH_ENTRY_VEHICLE);
```

Comment:
This message is not DEATH_MATCH related anymore but a generic limits_decider message.

Please follow message naming convention according to the other deciders.